SwiftWisdom/Core/StandardLibrary/Numbers/Bool+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Bool+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension Bool {
+    public init(ip_string: String) {
+        let cleaned = ip_string.lowercaseString
+        let truePrefixes = [
+            "t",
+            "y",
+            "1",
+        ]
+        
+        self = truePrefixes.contains { cleaned.hasPrefix($0) }
```

Comment:
This is how it was done in `Foundation`:

![screen shot 2016-01-20 at 12 01 10 am](https://cloud.githubusercontent.com/assets/5750489/12440371/f4b369bc-bf08-11e5-9690-679012802e52.png)

I think as long as a user is expecting to get a bool from a string, it should function as expected.